wolfssl/wolfcrypt/logging.h

```diff
@@ -26,11 +26,95 @@
 
 /* submitted by eof */
 
+/*
+ * Enable wolfSSL debugging with DEBUG_WOLFSSL
+ *
+ * Certificate debugging is a subset of DEBUG_WOLFSSL but can be enabled
+ * exclusively with WOLFSSL_DEBUG_CERTS.
+ *
+ * When DEBUG_WOLFSSL is enabled, but the subset of certificate debugging
+ * is not desired, it can disabled with NO_WOLFSSL_DEBUG_CERTS
+ *
+ * ****************************************************************************
+ * Message / printf debugging
+ * ****************************************************************************
+ *
+ * WOLFSSL_DEBUG_PRINTF_FN(...)
+ *  A buffer-less, non-truncating debug message renderer.
```

Comment:
`WOLFSSL_DEBUG_PRINTF()` is the utility macro.  `WOLFSSL_DEBUG_PRINTF_FN` is used to supply an override definition of the target platform's `printf`-like function, and it is not function-like:
```
#ifdef WOLFSSL_DEBUG_PRINTF_FN
    /* user-supplied definition */
#elif defined(ARDUINO)
    /* ARDUINO only has print and sprintf, no printf. */
#elif defined(WOLFSSL_LOG_PRINTF) || defined(WOLFSSL_DEOS)
    #define WOLFSSL_DEBUG_PRINTF_FN printf
[...]
```

Reply:
Ah, excellent. I'll update the documentation comments in logging.h - thanks for the clarification.